### PR TITLE
docs(readme): add usage part

### PR DESCRIPTION
**Pull request by antleblanc**
Enhance `README.md` file by adding a `Usage` part.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,20 @@
...
+## Usage
+
+```js
+import 'angular';
+import translateAsyncLoader from '@ovh-ux/translate-async-loader';
+
+angular
+  .module('myApp', [
+    …,
+    translateAsyncLoader;
```
Comment (jleveugle) [inline README.md]: `;` -> `,`

Comment (jleveugle) [inline README.md]:
```suggestion
    translateAsyncLoader,
```

File: README.md [hunk 2]
```diff
@@ -10,6 +10,20 @@
...
 
+## Usage
+
+```js
+import 'angular';
+import translateAsyncLoader from '@ovh-ux/translate-async-loader';
+
+angular
+  .module('myApp', [
+    …,
```
Comment (jleveugle) [inline README.md]: Can you remove `...` ?

**Comment by antleblanc** [comment]
@jleveugle I applied all of your suggestions.

File: README.md (hunk 1 above)
Comment (FredericEspiau) [reply README.md]: Don't your `eslint` detects this ?

**Pull request merged**